backend/config/validator/validation.yaml

```diff
@@ -219,3 +219,10 @@ App\Request\Order\OrderUpdateCaptainOrderCostRequest:
             - NotBlank: ~
         captainOrderCost:
             - NotBlank: ~
+
+App\Request\Order\OrderUpdateCaptainArrivedRequest:
+    properties:
+        id:
+            - NotBlank: ~
+        isCaptainArrived:
```

Comment:
When we pass "false" for isCaptainArrived, the field would still blank according to the validator.